src/runner/index.js

```diff
@@ -266,7 +266,7 @@ export default class Runner extends EventEmitter {
             .then(() => {
                 this._validateRunOptions();
 
-                return this.bootstrapper.createRunnableConfiguration();
+                return this.bootstrapper.createRunnableConfiguration({ disableTestSyntaxValidation });
```

Comment:
I propose to pass `disableTestSyntaxValidation` option via `this.bootstrapper.disableTestSyntaxValidation` as other bootstapper-related options such as `browsers`, `reporters` and etc.